pkg/iris/api_test.go

```diff
@@ -26,7 +27,7 @@ func TestGetConfiguration200(t *testing.T) {
 	defer server.Close()
 
 	// Use Client & URL from our local test server
-	api := NewApi(server.Client(), "kind-test", server.URL, "token")
+	api := services.NewApi(server.Client(), "kind-test", server.URL, "token")
```

Comment:
if we move API to the services package, then we should rename it to IrisApi

Reply:
renamed API as IrisApi and all usage